browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.cc

```diff
@@ -0,0 +1,135 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.h"
+
+#include <utility>
+
+#include "base/check.h"
+#include "brave/browser/ui/views/frame/vertical_tab_strip_region_view.h"
+#include "chrome/browser/ui/views/frame/browser_view.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/views/layout/fill_layout.h"
+#include "ui/views/widget/widget.h"
+
+#if defined(USE_AURA)
+#include "ui/views/view_constants_aura.h"
+#endif
+
+// static
+void VerticalTabStripWidgetDelegateView::Create(BrowserView* browser_view,
+                                                views::View* host_view) {
+  DCHECK(browser_view->GetWidget());
+
+  views::Widget::InitParams params;
+  params.delegate =
+      new VerticalTabStripWidgetDelegateView(browser_view, host_view);
+  params.parent = browser_view->GetWidget()->GetNativeView();
+  params.type = views::Widget::InitParams::TYPE_CONTROL;
+  // We need this to pass the key events to the top level widget. i.e. we should
+  // not get focus.
+  params.activatable = views::Widget::InitParams::Activatable::kNo;
```

Comment:
I'm not a views expert at all, but just checking whether this has any accessibility side effects?

Reply:
that's a really good point! Fortunately, when a child widget doesn't get focus, all key events will be handled by the top most widget. On the other hands, if the child widget gets a focus, the top most widget can't get key strokes and it will break all our shortcuts. So, this is rather to guarantee that we can support the shortcuts.

One thing I didn't test was that focus traversal feature, which advances focus to a next candidate view via F6 on Windows. And luckily it works well with this patch :) Thank you for pointing out this!